knodle/labeler/CheXpert/stages/finetuning.py

```diff
@@ -0,0 +1,99 @@
+"""Define mention classifier class."""
+import logging
+from negbio.pipeline import parse, ptb2ud, negdetect
+from negbio.neg import semgraph, propagator, neg_detector
+from negbio import ngrex
+import networkx as nx
+from .utils import *
+
+
+class ModifiedDetector(neg_detector.Detector):
+    """Child class of NegBio Detector class.
+
+    Overrides parent methods __init__, detect, and match_uncertainty.
+    """
+    def __init__(self, config: Type[ChexpertConfig]):
```

Comment:
brief parameter description here and in \_\_init\_\_ in other functions would be good